packages/vite/src/node/__tests__/utils.spec.ts

```diff
@@ -14,11 +15,32 @@ describe('injectQuery', () => {
     // this test will work incorrectly on unix systems
     test('normalize windows path', () => {
       expect(injectQuery('C:\\User\\Vite\\Project', 'direct')).toEqual(
-        'C:/User/Vite/Project?direct'
+        'C:\\User\\Vite\\Project?direct'
       )
     })
```

Comment:
I'm not sure but maybe we need to normalize path.
It seems related to #2435, #2614 but Idk whether this is needed.

Reply:
It can to exec `normalizePath` it will reducing '..' and '.' parts. But now it will lose ` When multiple slashes are found, they're replaced by a single one; when the path contains a trailing slash, it is preserved. On Windows backslashes are used.` this feature.